spec/async/barrier_spec.rb

```diff
@@ -1,17 +1,17 @@
 # frozen_string_literal: true
 
 # Copyright, 2019, by Samuel G. D. Williams. <http://www.codeotaku.com>
-# 
+#
```

Comment:
What would you say if I start slowly introducing Rubocop?
I've already tried, got about 7.5k(!) issues, .rubocop_todo.yml is about 700 lines long.

I could start with simple cases likes this, missing newlines at the end of files or enforcing tabs for indentation(no need to fix, code is already properly formatted afair)

Reply:
haha, I misread that as 75k issues lol.

When I tried RuboCop a while ago, IIRC it did not support tabs for indentation.

However, I believe that changed recently.

I'd be okay with introducing RuboCop but it would probably need to be the opposite way for me personally, it would need to disable every rule by default and we selectively enable ones that make sense. It would probably just be a small subset and if I did go in that direction I'd want to replicate it to other projects too.